duckduckgo/macos-browser
# Tespach/phishing detection tests

Task/Issue URL: https://app.asana.com/0/0/1208196336229421/f
Tech Design URL:
CC:

**Description**:
Implement test cases for phishing detection error page, tab extension, and privacy dashboard.

**Steps to test this PR**:
1. Run the UnitTests + IntegrationTests
2. Visit https://privacy-test-pages.site/security/badware/phishing.html
3. Ensure warning is thrown
4. Click through warning



**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
